src/app/page.tsx

```diff
@@ -85,32 +84,25 @@ export default function Home() {
         {isLoading ? (
           <p className="text-white">Loading...</p>
         ) : (
-          <>
+          <div>
             <div className="flex items-center py-6 px-4 w-[723px] h-[57px] mb-4">
               <span className="text-white text-[24px] w-[250px]">
                 Company Name
               </span>
-
               <span className="text-white text-[24px] w-[200px] text-center">
                 Evaluation Date
               </span>
-
               <span className="text-white text-[24px] ml-auto">Score</span>
             </div>
 
             {filteredData.length > 0 ? (
               filteredData.map((company) => (
-                <CompanyCard
-                  key={company.id}
-                  name={company.name}
-                  date={formatDate(company.date)}
-                  score={company.score}
-                />
+                <CompanyCard key={company.id} company={company} />
               ))
             ) : (
               <p className="text-white mt-4">No companies found</p>
```

Comment:
Center the 'No Companies Found' message

```suggestion
              <p className="text-white text-center mt-6">No companies found</p>
```